src/BeginRestore.js

```diff
@@ -128,6 +130,9 @@ export class BeginRestore extends Component {
                 <Row>
                     {RequiredBoolean(this, "Write files atomically", "writeFilesAtomically")}
                 </Row>
+                <Row>
+                    {RequiredBoolean(this, "Write files sparsely", "writeFilesSparsely")}
```

Comment:
I think "sparse files" is more obvious here, so "Write sparse files". Also the second argument should be the name of the JSON field, so "sparse".

See https://github.com/kopia/htmlui/pull/57#issuecomment-1155857150 for testing instructions.

Reply:
That makes sense, thanks for the feedback and test instructions. Iterating

Reply:
This needs to match JSON tag in `restore.FilesystemOutput`:

```
	// Sparse when set to true causes the restored files to be sparse.
	Sparse bool `json:"sparse"`
```

Reply:
Gotcha, so what does this line do:
```diff
req.fsOutput = {
  ...
  writeFilesAtomically: this.state.writeFilesAtomically
+ sparse: this.state.writeFilesSparsely,
}
```

Reply:
ah, got it

Reply:
this is a new feature so not really breaking, i think

Reply:
Implemented this here: https://github.com/kopia/kopia/pull/2095. I'll follow up with changes to the UI once the above is merged.